src/PaymentMethods/iDeal/iDeal.php

```diff
@@ -57,58 +58,26 @@ public function payRemainder(?Model $model = null): TransactionResponse
     }
 
     /**
-     * @return \string[][]
+     * @return \mixed
      * @throws \Buckaroo\Exceptions\BuckarooException
      */
-    public function issuers(): array
+    public function issuers() : mixed 
     {
-        $issuers = [
-            [
-                'id' => 'ABNANL2A',
-                'name' => 'ABN AMRO'
-            ],
-            [
-                'id' => 'ASNBNL21',
-                'name' => 'ASN Bank'
-            ],
-            [
-                'id' => 'BUNQNL2A',
-                'name' => 'bunq'
-            ],
-            [
-                'id' => 'INGBNL2A',
-                'name' => 'ING'
-            ],
-            [
-                'id'    => 'KNABNL2H',
-                'name'  => 'Knab'
-            ],
-            [
-                'id' => 'RABONL2U',
-                'name' => 'Rabobank'
-            ],
-            [
-                'id' => 'RBRBNL21',
-                'name' => 'RegioBank'
-            ],
-            [
-                'id' => 'REVOLT21',
-                'name' => 'Revolut'
-            ],
-            [
-                'id' => 'SNSBNL2A',
-                'name' => 'SNS Bank'
-            ],
-            [
-                'id' => 'TRIONL2U',
-                'name' => 'Triodos Bank'
-            ],
-            [
-                'id' => 'FVLBNL22',
-                'name' => 'Van Lanschot'
-            ]
-        ];
+        $request = new TransactionRequest;
 
-        return $issuers;
-    }
+        try {
+            $response = $this->client->specification($request, 'ideal', 2);
+        }catch(BuckarooException $e){
+            return false;
+        }
+
+        $issuersData = $response['Actions']['0']['RequestParameters'][0]['ListItemDescriptions'];
```

Comment:
I think it's better to check the content of this array. Use issets and counter to confirm the content. Otherwise, it will spit out "is not set" exceptions if the response changes.

Reply:
Added a new commit with the suggested checks. Thanks :)